ciao-cli/image.go

```diff
@@ -97,18 +87,11 @@ var imageCommand = &command{
 }
 
 type imageAddCommand struct {
-	Flag            flag.FlagSet
-	name            string
-	id              string
-	containerFormat imageContainerFormat
-	diskFormat      imageDiskFormat
-	minDiskSize     int
```

Comment:
Image implies read-only.  Once uploaded, we know the actual size of the image and done.  An image minimum size seems unuseful to me.